plugins/tvshows/controller.js

```diff
@@ -1,24 +1,26 @@
 function TVShows($scope, $http, $interval) {
 
     getTVShows()
-    $interval(getTVShows, config.tvshows.refreshInterval * 60000 || 7200000)
+    $interval(getTVShows, (config.tvshows ? config.tvshows.refreshInterval * 60000 : 7200000));
```

Comment:
Oh, duh! Good catch. We probably need to fix this for the rest of the services as well.